sdk/openai/Azure.AI.OpenAI/tests/Utils/ModelReaderWriterConverter.cs

```diff
@@ -46,7 +46,7 @@ public InnerModelReaderWriterConverter()
             /// <inheritdoc />
             public override T Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
```

Comment:
Shouldn't we update this to be nullable? Or should the entire test class have nullability disabled? /cc @joseharriaga